browser-test/src/application_review.test.ts

```diff
@@ -21,15 +21,15 @@ describe('normal application flow', () => {
     await adminQuestions.addNumberQuestion('number-q');
     await adminQuestions.addTextQuestion('text-q');
     await adminQuestions.addRadioButtonQuestion('radio-q', ['one', 'two', 'three']);
-    await adminQuestions.addStaticQuestion('static-q');
+    await adminQuestions.addStaticQuestion('q-static');

```

Comment:
Just out of curiosity, was this name change significant to getting the test to pass? Super minor nit, but it departs from the naming convention of all the other questions.

Reply:
Seems like we need the two names for test static questions to be different enough so that expectActiveQuestionExist recognizes both. Seems like the issue before was that we had 'static-q' and 'second-static-q', so sometimes the test would look for 'static-q' and find 'second-static-q' and think it passed, but that is a faulty pass.

Let's name questions something more descriptive, or add a comment to explain why they're named like this. We want a new reader of the code to be able to easily why we did something -- it could help them solve bugs and write tests in the future. 

I think 'first-static-q' and 'second-static-q' should be enough to resolve flakiness here.  What do you think?

Reply:
Yeah that describes it perfectly!! Agreed 'first-static-q' and 'second-static-q' are good names to solve the issue and stick to the convention. Updated, thanks!